### PR TITLE
made changes to reflect kubectl functionality installation

### PR DESCRIPTION
**Pull request by Supekar**
Made changes to reflect 'appconfig.tgz' file location and new image name for additional kubectl functionality

### CODE REVIEW

File: deploy/example_catalog/cr-app-tensorflow-notebook.json [hunk 1]
```diff
@@ -45,15 +59,20 @@
                 }
             }
         ],
-        "defaultConfigPackage": null,
+        "defaultConfigPackage": {
```
Comment (kmathur2) [inline deploy/example_catalog/cr-app-tensorflow-notebook.json]: we don't need this. Appconfig should be baked into image

File: deploy/example_catalog/cr-app-tensorflow-notebook.json [hunk 2]
```diff
@@ -2,36 +2,50 @@
...
             "selectedRoles": [
-                "controller"
+                "controller",
+                "worker"
             ]
         },
         "label": {
-            "name": "TensorFlow + Jupyter",
-            "description": "TensorFlow GPU with Jupyter notebook"
+            "name": "Tensorflow distributed 1.9 on CPU with jupyter notebook & centos7x",
```
Comment (kmathur2) [inline deploy/example_catalog/cr-app-tensorflow-notebook.json]: looks like you made changes to old version, you should use this json as base and make your changes

File: deploy/example_catalog/cr-app-tensorflow-notebook.json [hunk 3]
```diff
@@ -45,15 +59,20 @@
                 }
             }
         ],
-        "defaultConfigPackage": null,
+        "defaultConfigPackage": {
+            "packageURL" : "http://16.143.21.22/thirdparty/k8sapps/notebook/appconfig-1.8.tgz"
+        },
         "roles": [
             {
+                "imageRepoTag": "docker.io/rsupekar/kd-notebook:latest",
```
Comment (kmathur2) [inline deploy/example_catalog/cr-app-tensorflow-notebook.json]: What was the base for this docker image ? We need to use the docker image that saurabh has.

File: deploy/example_catalog/cr-app-tensorflow-notebook.json [hunk 4]
```diff
@@ -45,15 +59,20 @@
...
                 "cardinality": "1",
-                "id": "controller",
-                "minResources":{
-                    "nvidia.com/gpu": "1"
-                }
+                "id": "controller"
+            },
+            {
+                "imageRepoTag": "docker.io/rsupekar/kd-notebook:latest",
```
Comment (kmathur2) [inline deploy/example_catalog/cr-app-tensorflow-notebook.json]: same image, same comment

File: deploy/example_clusters/cr-cluster-tensorflow-notebook.yaml [hunk 5]
```diff
@@ -1,17 +1,19 @@
...
-        cpu: "2"
-        nvidia.com/gpu: "1"        
+        cpu: "1"
       limits:
         memory: "4Gi"
-        cpu: "2"
-        nvidia.com/gpu: "1"
\ No newline at end of file
+        cpu: "1"
+
```
Comment (kmathur2) [inline deploy/example_clusters/cr-cluster-tensorflow-notebook.yaml]: we do not want to remove nvidia gpu

File: deploy/example_catalog/cr-app-tensorflow-notebook.json [hunk 6]
```diff
@@ -2,36 +2,50 @@
     "apiVersion": "kubedirector.hpe.com/v1beta1",
     "kind": "KubeDirectorApp",
     "metadata": {
-        "name": "ml-jupyter-notebook",
-        "annotations": {
-            "token":"execute this command to get the authentication token 'kubectl exec <pod-name> jupyter notebook list' and use this token in Jupyter notebook."
-        }
+        "name" : "jupyter-notebook"
     },
```
Comment (kmathur2) [inline deploy/example_catalog/cr-app-tensorflow-notebook.json]: @saurabh-jogalekar This looks like your master branch and not MLOPs right ?

Comment (saurabh-jogalekar) [reply deploy/example_catalog/cr-app-tensorflow-notebook.json]: @kmathur2 @Supekar
Yes, this is not MLOps image. Please refer to the latest commit for MLOps Apps.
Link: https://github.com/saurabh-jogalekar/kubedirector/commit/25319725d3af3491ea3c7e73c06ac203666f75f1

Comment (kmathur2) [reply deploy/example_catalog/cr-app-tensorflow-notebook.json]: this PR should be against MLOps branch

Comment (saurabh-jogalekar) [reply deploy/example_catalog/cr-app-tensorflow-notebook.json]: Look like it is the correct branch but incorrect file name.

Comment (saurabh-jogalekar) [reply deploy/example_catalog/cr-app-tensorflow-notebook.json]: This image is of the tensorflow+jupyter app that is currently used in HPECP. It is directly derived from tensorflow docker hub. The app we want the changes to go in would be [this](https://github.com/saurabh-jogalekar/kubedirector/blob/mlops-lift-and-shift/deploy/example_catalog/cr-app-tensorflowcpu.json)

Comment (Supekar) [reply deploy/example_catalog/cr-app-tensorflow-notebook.json]: will re do the changes on the correct file

File: deploy/example_catalog/cr-app-tensorflowcpu.json [hunk 7]
```diff
@@ -53,11 +53,11 @@
             }
         ],
         "defaultConfigPackage": {
-            "packageURL" : "http://16.143.21.166/thirdparty/k8sapps/notebook/appconfig-1.8.tgz"
+            "packageURL" : "http://16.143.21.22/thirdparty/k8sapps/notebook/appconfig-1.8.tgz"
```
Comment (kmathur2) [inline deploy/example_catalog/cr-app-tensorflowcpu.json]: This should be removed, rebuild the image with app config as part of the image. appconfig can live under /opt/configscript, then replace this path with "/opt/configscript/appconfig-1.8.tgz

Comment (Supekar) [reply deploy/example_catalog/cr-app-tensorflowcpu.json]: Noted. Will do.

**Review by kmathur2 — APPROVED** [review]

**Comment by kmathur2** [comment]
Looks good 👍

**Pull request merged**